Whisky/Views/Bottle/BottleRenameView.swift

```diff
@@ -21,48 +21,43 @@ import WhiskyKit
 
 struct BottleRenameView: View {
     let bottle: Bottle
-    @State var newBottleName: String = ""
-    @State var nameValid: Bool = false
     @Binding var name: String
-    @Environment(\.dismiss) var dismiss
+
+    @State private var newBottleName: String = ""
+    @Environment(\.dismiss) private var dismiss
 
     var body: some View {
-        VStack {
```

Comment:
Could you consider refactoring this popup component into something in `common`? You don't have too but since you're changing it, it may cause other views to look inconsistent -- Also, please provide a screenshot if possible.

Reply:
Added a generic `RenameView`. Added Screenshots. The changes are so they look more consistent and more like macOS Default sheets.

Also see #659 where the attempt was already made to give them a unified look, but those two were still missing.

Reply:
It looks like that is merged. What I meant is have like a popup component in common.

Reply:
Hmm... you mean like a generic component for sheets/popups in general?
I mean its possible, but not sure if its worth the effort since the basics are simply like this

```swift
NavigationStack {
    // popup content
    .navigationTitle(/* popup title*/)
    .toolbar {
        // actions
    }
}
```

Reply:
If it's a simple as that we might not have to. Sorry, I didn't see it very clearly.